### ISSUE TITLE
Error handling Unicode messages

### ISSUE THREAD

**Issue by krepver** [reo7sp/tgbot-cpp#21]
If i send bot message like "привет", telegram return json like this

``` json
{
 "ok":true,
 "result":
 [{
  "update_id":97062,
  "message":
  {
   "message_id":61,
   "from":
   {
    "id":96202,
    "first_name":"Aleksei",
    "last_name":"Aleshin",
    "username":"krepver"
   },
   "chat":
   {
    "id":96202,
    "first_name":"Aleksei",
    "last_name":"Aleshin",
    "username":"krepver",
    "type":"private"
   },
   "date":1460646730,
   "text":"\u041f\u0440\u0438\u0432\u0435\u0442"
  }
 }]
}
```

But `message->text` field is "\177\177\177\177\177\177"
![wr_message](https://cloud.githubusercontent.com/assets/1073706/14533449/3117b59c-0276-11e6-9d07-a7948dea2129.png)

**Comment by aadeg** [comment]
Hi, tgbot-cpp uses Boost library to parse json responses. According this ticket https://svn.boost.org/trac/boost/ticket/8883 Boost has a bug with unicode parsing.
The bug has been fixed with boost 1.59 (http://www.boost.org/users/history/version_1_59_0.html), so you only need to update boost to the recent version.

**Comment by krepver** [comment]
Thank you, I will try it.
Would be nice to have a minimal version requirement in readme.

**Comment by In-line** [comment]
Solved my problem. +1

**Comment by reo7sp** [comment]
I think it's solved by #33
